app/static/js/app.js

```diff
@@ -247,15 +250,50 @@ function onKeyDown(evt) {
 }
 
 function sendMouseEvent(evt) {
-  const boundingRect = evt.target.getBoundingClientRect();
-  const cursorX = Math.max(0, evt.clientX - boundingRect.left);
-  const cursorY = Math.max(0, evt.clientY - boundingRect.top);
+  // Ensure that e.g. mouse drags don't attempt to drag the image on the screen.
+  evt.preventDefault();
```

Comment:
I want to be conservative about calling `evt.preventDefault` because it has the potential to mess with other tools, especially accessibility functionality.

I think the only mouse events that need `preventDefault` are `mousemove` and `touchmove`, correct? Could we shift the code around so that those handlers call it explicitly and the other events don't?